man/systemd.netdev.xml

```diff
@@ -911,6 +916,23 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Sequence=</varname></term>
```

Comment:
"Sequence" is a bit too generic. Why not call this "SerializeTunneledPackets=" ?